osf/models/admin_log_entry.py

```diff
@@ -1,10 +1,12 @@
 from django.contrib.admin.models import LogEntry, LogEntryManager
 
 UNKNOWN = 0
-ACCEPT_PREREG = 10
-REJECT_PREREG = 11
-COMMENT_PREREG = 12
-CHECKOUT_CHECKUP = 13
+
+# Depreciated, used for Prereg Challenge
+# ACCEPT_PREREG = 10
+# REJECT_PREREG = 11
+# COMMENT_PREREG = 12
```

Comment:
Why comment instead of remove?

Reply:
because the numbers are still in the prod logs, if someone is reading the prod db they'd have no idea what 10,11,12 indicate.